### DISCUSSION TITLE
2005/05/12/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#26] [Announcements]
# 2005/05/12/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-7

https://blog.qizhen.xyz/2005/05/12/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-7

**Comment by ruanqizhen** [comment]
煮咖啡的鱼
2005年05月13日 11:24 编辑
踩一脚。　

回复
lai
2005年05月13日 10:23 编辑
HI，恭喜成为我家第1000位来访者。欢迎有空常来哦~~~~~~~~~~~

回复
Qizhen
2005年05月13日 06:25 编辑
计数器怎么弄的，看左边《Space 技术大全》，里面都有了。

回复
Unknown
2005年05月13日 04:34 编辑
我的弄好了,有歌了,真谢谢你了,把你加好友拉

回复
YAKI
2005年05月13日 02:06 编辑
戴眼镜好啊，不容易进灰,还挡风哦~~呵呵~~~YAKI

回复
Huajun
2005年05月12日 19:24 编辑
计数器怎么弄的？

回复
shelly
2005年05月12日 15:22 编辑
对不起啊，刚刚才发现，你的联系人里的KAREN不是我在英国的表妹，不过名字一样，呵呵，不好意思啊。我表妹的是http://spaces.msn.com/members/karenlee1122

回复
shelly
2005年05月12日 15:02 编辑
哦，原来是这样，谢谢你的光顾，也很高兴认识你，对了，你联系人里那个叫KAREN的是我表妹，我们现在都不在中国，不过很想家，中国现在一定很热吧。。。6月份想回国，好期待哦。。。

回复
